CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+
+## Core CIPs
+For Core CIPs, given that they require Celo’s client implementation to be considered **Final** (see “CIPs Process” below), the blockchain author will need to either provide an implementation for the celo-blockchain client or convince clients to implement your CIP.
+
+The best way to get client implementers to review your CIP is to present it on a Celo AllCoreDevs call. You can request to do so by posting a comment linking your CIP on an agenda issue for an upcoming call on the Github repository.
+
+The Celo AllCoreDevs call serve as a way for client implementers and stakeholders to do two things. First, to discuss the technical merits of CIPs. Second, to coordinate CIP implementation for network upgrades. 
+
+These calls generally result in a “[rough consensus](https://en.wikipedia.org/wiki/Rough_consensus)” around what CIPs should be implemented. Rough consensus, first used by IETF to describe how working groups would proceed, rests on the assumptions that CIPs are not contentious enough to cause a network split and that they are technically sound. It also states that 51% of working group voting on a specific action isn't rough consensus, while 99% is better than rough consensus.
+
+⚠️ The CIPs process and AllCoreDevs call were not designed to address contentious non-technical issues, but, due to the lack of other ways to address these, often end up entangled in them. This puts the burden on core developers to try and gauge community sentiment, which hinders the technical coordination function of CIPs and AllCoreDevs calls. If you are shepherding a CIP, you can make the process of building community consensus easier by making sure that the Github issue thread for your CIP includes or links to as much of the community discussion as possible and that various stakeholders are well-represented.
+
+_In short, your role as the champion is to write the CIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea._
+
+## CIP Process
+When we talk about the community from the perspective of the CIP Process, we are talking about all the validators, stakeholders, core-developers, dapp developers, volunteers and active participants involved in the CIP Process.
+
+Rough Consensus, as mentioned in the previous section, is used to measure community sentiment about a particular proposal. Rough Consensus is measured mostly on the All-Core Devs calls, but it can also happen outside of those calls on Github or Celo Forum comments. Rough Consensus will be determined by the Hardfork Coordinator and All-Core Dev Call organizer based on feedback of the network participants during the call. 
```

Comment:
let's find an off-chain async polling mechanism rather than do this on a call.

Reply:
Due to avoiding sybil attacks, IETF Working Groups proposing Rough Consensus as a measure during a call has been pretty successful in the past in a lot of the standards being specified.

If we do off-chain polling like say a poll on Discord about people's opinions on somethings, we heavily risk being sybil attacked by someone pushing for a particular direction. I've seen it happy many times, a sudden stream of sockpuppets entering a chat room, mostly voting and preaching a particular proposal.

What we can do is add off-chain polling as a secondary signal, but I am very hesitant to use it to inform consensus.

Consensus has to be rough at least for upcoming hardforks. If there are any disagreements due to majority-minority votes, we are risking network splits.

Reply:
the call is an important part of Ethereum's process, in my opinion. It ensures that devs and stakeholders to show up and advocate for their point of view, and that decisions are made in a fixed cadence

Reply:
how do they deal with timezones, meeting efficiency, etc? there are certainly individuals who have a lot to contribute but whose English proficiency makes it harder for them to participate in calls than async. howo you feel we can accommodate this and also make this a productive call?

Reply:
3PM UTC is normally best time that makes everyone happy since it includes most of the user timezones. The calls are mostly for technical specifications and improvements to the Celo Protocol. They will be recorded, they can be transcribed and translated via Google Translate later if needed to allow everyone to review them. Community calls are a huge help for technical specifications of the protocol since debates need a natural way to flow in regards to updating of the client while measuring rough consensus.